RU: add links to redis university courses/learning paths (#1093)
redis/docs: content/operate/rs/databases/create.md

    See [Configuration settings]({{< relref "/operate/rs/databases/configure#config-settings" >}}) for more information about each setting.

1. Select **Create**.

## Continue learning with Redis University

> See the [Create a database on Redis Software course](https://university.redis.io/course/cvj3ntarrg8roa) for modules.

Reviewer: ```suggestion
See the [Create a database on Redis Software course](https://university.redis.io/course/cvj3ntarrg8roa) to learn more.
```
